### PR TITLE
FE-1079 - Within table rows, <Button></Button> components should have icons and use the flat, gray color

### PR DESCRIPTION
**Pull request by nicklemmon**
[FE-1079](https://jira.int.messagesystems.com/browse/FE-1079)

### What Changed
- Adds new `"minimal"` variant to the `Button` component
- Increases `z-index` by 1 for the OG Modal - this helps with overlay problem when opening a modal via a `Popover`
- Adds screen reader only content within empty table headings
- Updates buttons within tables in the app:
  + When more than one `Button` is present, uses the `ActionPopover` component
  + When only one `Button` is present, uses the `"minimal"` variant paired with an icon on the righthand side
- Updates tests accordingly

### How To Test
- Verify all buttons within tables were updated according to the a/c
- Verify that existing behavior within these tables continues to function as expected

### To Do
- [ ] Incorporate feedback

### CODE REVIEW

**Comment by bkemper** [comment]
The vertical alignment of text and chevron looks off.

Screen Shot 2020-06-04 at 4 33 38 PM

**Comment by bkemper** [comment]
I know my preference doesn't matter, but I prefer the blacklist watch list buttons.

Screen Shot 2020-06-04 at 4 37 49 PM
Screen Shot 2020-06-04 at 4 38 19 PM

File: .circleci/config.yml [hunk 1]
```diff
@@ -74,15 +74,15 @@ jobs:
 
   'Integration Testing':
     <<: *defaults
-    parallelism: 4
+    parallelism: 1
```
Comment (bkemper) [inline .circleci/config.yml:77]: huh?

Comment (bkemper) [reply .circleci/config.yml:77]: I'm assuming this is to test the broken hibana test?

File: cypress/tests/integration/events/listPage.spec.js [hunk 2]
```diff
@@ -472,10 +472,12 @@ describe('The events page', () => {
 
       cy.findAllByText('View Details')
```
Comment (bkemper) [inline cypress/tests/integration/events/listPage.spec.js:473]: Should we make a `cy.findIconLinkByText('View Details')`

File: src/components/actionPopover/ActionPopover.js [hunk 3]
```diff
@@ -13,8 +13,10 @@ const ActionPopover = ({ actions }) => {
         id={uniqueId}
         left
         trigger={
-          <Button aria-controls={uniqueId} flat>
-            <MoreHoriz size={20} />
+          <Button aria-controls={uniqueId} variant="minimal">
+            <MoreHoriz />
+
+            <ScreenReaderOnly>Open Menu</ScreenReaderOnly>
```
Comment (bkemper) [inline src/components/actionPopover/ActionPopover.js:19]: 👏

File: src/components/collection/SortLabelHibana.module.scss [hunk 4]
```diff
@@ -4,6 +4,7 @@
 .SortLabel {
   display: flex;
   align-items: center;
+  background: none;
```
Comment (bkemper) [inline src/components/collection/SortLabelHibana.module.scss:7]: 👏

File: src/components/globalAlert/GlobalAlert.module.scss [hunk 5]
```diff
@@ -5,7 +5,7 @@
   top: rem(75);
   right: spacing();
 
-  z-index: z-index(overlay);
+  z-index: z-index(overlay) + 1;
```
Comment (bkemper) [inline src/components/globalAlert/GlobalAlert.module.scss:8]: Should this be eventually moved back to matchbox/design-tokens?

File: src/pages/trackingDomains/components/TrackingDomainRow.js [hunk 6]
```diff
@@ -133,37 +116,53 @@ export class TrackingDomainRow extends Component {
...
+                    className={styles.Tag}
+                    id={subaccountId}
+                    isDefault={!!subaccountId && isDefault}
+                  />
+                )}
+              </div>
+            </Grid.Column>
+            <Grid.Column xs={12} md={4}>
+              <ActionPopover
+                actions={[
```
Comment (bkemper) [inline src/pages/trackingDomains/components/TrackingDomainRow.js:142]: Screen Shot 2020-06-04 at 11 29 59 PM

File: src/components/globalAlert/GlobalAlert.module.scss (hunk 5 above)
Comment (nicklemmon) [reply src/components/globalAlert/GlobalAlert.module.scss:8]: Not sure what you mean? Making a new design token?

These functions are from OG Matchbox.

File: cypress/tests/integration/events/listPage.spec.js (hunk 2 above)
Comment (nicklemmon) [reply cypress/tests/integration/events/listPage.spec.js:473]: I'm not sure? `findAll*` works just fine right? The force clicking is a little strange.

File: .circleci/config.yml (hunk 1 above)
Comment (nicklemmon) [reply .circleci/config.yml:77]: Yep, whoops. Let me re-introduce the right value.

**Comment by nicklemmon** [comment]
> I know my preference doesn't matter, but I prefer the blacklist watch list buttons.
> 
> Screen Shot 2020-06-04 at 4 37 49 PM
> 
> Screen Shot 2020-06-04 at 4 38 19 PM

I agree with you...

But I actually need to change this `Button` as well - looks like I missed those.

**Comment by nicklemmon** [comment]
> The vertical alignment of text and chevron looks off in OG, good in Hibana.
> 
> Screen Shot 2020-06-04 at 4 33 38 PM

These are basically two Matchbox components interacting with each other - so I think it should be resolved there?

File: src/components/globalAlert/GlobalAlert.module.scss (hunk 5 above)
Comment (bkemper) [reply src/components/globalAlert/GlobalAlert.module.scss:8]: Oops, I was thinking `overlay` was a design token and needed to incremented.

File: src/components/modals/DeleteModal.js [hunk 7]
```diff
@@ -8,7 +8,7 @@ export default function DeleteModal({ onDelete, deleting, ...props }) {
       confirmVerb="Delete"
       onConfirm={onDelete}
       confirmVariant="destructive"
-      cancelVariant="destructive-secondary"
```
Comment (bkemper) [inline src/components/modals/DeleteModal.js:11]: 👍

**Pull request merged**